### PR TITLE
Addon Manager: Add MakerWorkbench icon

### PR DESCRIPTION
**Pull request by davidmubernal**
Thank you for creating a pull request to contribute to FreeCAD! To ease integration, we ask you to conform to the following items. Pull requests which don't satisfy all the items below might be rejected. If you are in doubt with any of the items below, don't hesitate to ask for help in the [FreeCAD forum](https://forum.freecadweb.org/viewforum.php?f=10)!

- [ ]  Your pull request is confined strictly to a single module. That is, all the files changed by your pull request are either in `App`, `Base`, `Gui` or one of the `Mod` subfolders. If you need to make changes in several locations, make several pull requests and wait for the first one to be merged before submitting the next ones
- [ ]  In case your pull request does more than just fixing small bugs, make sure you discussed your ideas with other developers on the FreeCAD forum
- [ ]  Your branch is [rebased](https://git-scm.com/docs/git-rebase) on latest master `git pull --rebase upstream master`
- [ ]  All FreeCAD unit tests are confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [ ]  All commit messages are [well-written](https://chris.beams.io/posts/git-commit/) ex: `Fixes typo in Draft Move command text`
- [ ]  Your pull request is well written and has a good description, and its title starts with the module name, ex: `Draft: Fixed typos`
- [ ]  Commit messages include `issue #<id>` or `fixes #<id>` where `<id>` is the [FreeCAD bug tracker issue number](https://freecadweb.org/wiki/tracker#GitHub_and_MantisBT) in case a particular commit solves or is related to an existing issue on the tracker. Ex: `Draft: fix typos - fixes #0004805`

And please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [0.19 Changelog Forum Thread](https://forum.freecadweb.org/viewtopic.php?f=10&t=34586).

---

### CODE REVIEW

File: src/Mod/AddonManager/Resources/AddonManager.qrc [hunk 1]
```diff
@@ -34,6 +34,7 @@
         <file>icons/LCInterlocking_workbench_icon.svg</file>
         <file>icons/Lithophane_workbench_icon.svg</file>
         <file>icons/Manipulator_workbench_icon.svg</file>
+        <file>icons/Maker_workbench_icon.svg</file>
```
Comment (luzpaz) [inline src/Mod/AddonManager/Resources/AddonManager.qrc:37]: this line should go before line 36

File: src/Mod/AddonManager/Resources/icons/Maker_workbench_icon.svg [hunk 2]
```diff
@@ -0,0 +1,359 @@
...
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
+
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
```
Comment (luzpaz) [inline src/Mod/AddonManager/Resources/icons/Maker_workbench_icon.svg:11]: Can you re-export this SVG as a `Plain SVG` instead as an inkscape SVG? (Should be available as an export option in Inkscape)

**Review by luzpaz — CHANGES_REQUESTED** [review]

**Comment by luzpaz** [comment]
Please squash down in to one commit, thanks

**Comment by davidmubernal** [comment]
Perfect, I will create another Pull Request to apply the changes and do it in one commit. (I have to learn more about GitHub)

**Comment by luzpaz** [comment]
Not necessary! Please read up on https://medium.com/@slamflipstrom/a-beginners-guide-to-squashing-commits-with-git-rebase-8185cf6e62ec

**Comment by luzpaz** [comment]
Please close as this was superseded by #4475

**Comment by luzpaz** [comment]
@davidmubernal please close this PR. thank you.

**Pull request closed without merging**